serverless.yml

```diff
@@ -67,30 +63,90 @@ resources:
       Type: AWS::EC2::VPC
       Properties:
         CidrBlock: '10.0.0.0/16'
-    BertlySubnetA:
+    BertlyElasticIp:
+      Type: AWS::EC2::EIP
+      Properties:
+        Domain: vpc
+    BertlyInternetGateway:
+      Type: AWS::EC2::InternetGateway
+    BertlyVPCGatewayAttachment:
+      Type: AWS::EC2::VPCGatewayAttachment
+      Properties:
+        VpcId:
+          Ref: BertlyVPC
+        InternetGatewayId:
+          Ref: BertlyInternetGateway
+    BertlyPublicRouteTable:
+      Type: AWS::EC2::RouteTable
+      Properties:
+        VpcId:
+          Ref: BertlyVPC
+    BertlyPublicRoute:
+      Type: AWS::EC2::Route
+      Properties:
+        RouteTableId:
+          Ref: BertlyPublicRouteTable
+        DestinationCidrBlock: 0.0.0.0/0
+        GatewayId:
+          Ref: BertlyInternetGateway
+    BertlyNatGateway:
+      Type: AWS::EC2::NatGateway
+      Properties:
+        AllocationId:
+          Fn::GetAtt: BertlyElasticIp.AllocationId
+        SubnetId:
+          Ref: BertlyPublicSubnetA
+    BertlyPrivateRouteTable:
+      Type: AWS::EC2::RouteTable
+      Properties:
+        VpcId:
+          Ref: BertlyVPC 
+    BertlyPrivateRoute:
+      Type: AWS::EC2::Route
+      Properties:
+        RouteTableId:
+          Ref: BertlyPrivateRouteTable
+        DestinationCidrBlock: 0.0.0.0/0
+        NatGatewayId:
+          Ref: BertlyNatGateway
+    BertlySubnetRouteTableAssociationPublicA:
+      Type: AWS::EC2::SubnetRouteTableAssociation
+      Properties:
+        SubnetId:
+          Ref: BertlyPublicSubnetA
+        RouteTableId:
+          Ref: BertlyPublicRouteTable
+    BertlyPublicSubnetA:
       DependsOn: BertlyVPC
       Type: AWS::EC2::Subnet
       Properties:
         VpcId:
           Ref: BertlyVPC
         AvailabilityZone: ${self:provider.region}a
-        CidrBlock: '10.0.0.0/24'
-    BertlySubnetB:
+        CidrBlock: '10.0.1.0/24'
+    BertlySubnetRouteTableAssociationPrivateA:
+      Type: AWS::EC2::SubnetRouteTableAssociation
+      Properties:
+        SubnetId:
+          Ref: BertlyPrivateSubnetA
+        RouteTableId:
+          Ref: BertlyPrivateRouteTable
+    BertlyPrivateSubnetA:
       DependsOn: BertlyVPC
       Type: AWS::EC2::Subnet
       Properties:
         VpcId:
           Ref: BertlyVPC
-        AvailabilityZone: ${self:provider.region}c
-        CidrBlock: '10.0.1.0/24'
-    BertlySubnetC:
+        AvailabilityZone: ${self:provider.region}a
+        CidrBlock: '10.0.2.0/24'
+    BertlyPrivateSubnetB:
       DependsOn: BertlyVPC
       Type: AWS::EC2::Subnet
       Properties:
         VpcId:
           Ref: BertlyVPC
-        AvailabilityZone: ${self:provider.region}d
-        CidrBlock: '10.0.2.0/24'
+        AvailabilityZone: ${self:provider.region}b
+        CidrBlock: '10.0.3.0/24'
```

Comment:
This is a subtle change. Can you explain the change from `2` to `3` in that octet?

Reply:
Nevermind! Got it. Because you have three subnets now.